src/Altinn.Apps/AppFrontend/react/shared/src/language/texts/en.ts

```diff
@@ -188,7 +188,7 @@ export function en() {
       is_sent: 'is submitted',
       receipt: 'Receipt',
       receiver: 'Receiver',
-      reference_number: 'Reference number:',
+      reference_number: 'Reference number',
       sender: 'Sender',
       sent_content: 'The following is submitted:',
```

Comment:
Consider removing the colon from the code instead or here as well so we are consistent in adding it either in the text resources or in the code